plugins/org.wso2.developerstudio.eclipse.distribution.project/src/org/wso2/developerstudio/eclipse/distribution/project/ui/wizard/DistributionProjectWizard.java

```diff
@@ -52,12 +52,14 @@
 		AbstractWSO2ProjectCreationWizard {

```

Comment:
Since you haven't done any changes for this class, do not add this class file in the commit. Use the wso2 formatter to format the class.